tests/testthat/test-table_with_settings_ui.R

```diff
@@ -74,7 +74,17 @@ testthat::test_that(
       name = "tws",
       variant = "app_driver_tws_ui",
     )
-    app_driver$wait_for_idle(timeout = longer_timeout)
+
+    # Wait for table content to be ready - poll until table content appears
+    max_attempts <- 10
+    for (i in seq_len(max_attempts)) {
+      app_driver$wait_for_idle(timeout = longer_timeout)
+      table_text <- app_driver$get_text("#table_with_settings-table_out_main")
+      if (nchar(table_text) > 0 && grepl("B: Placebo|C: Combination", table_text)) {
+        break
+      }
+      Sys.sleep(0.5)
+    }
```

Comment:
either we try 10 times, and wait 0.5 second after each attempt. or we increase idle time 4 times : P
```suggestion
    app_driver$wait_for_idle(timeout = longer_timeout * 4)
```